x-pack/test/apm_api_integration/tests/time_range_metadata/many_apm_server_versions.spec.ts

```diff
@@ -95,7 +90,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           });
 
           // @ts-expect-error
-          expect(res.hits.total.value).to.be(20);
+          expect(res.hits.total.value).to.be(10);
```

Comment:
There are no longer `serviceTransactionMetric` docs for the scenario to simulate APM v8.5.0. Therefore, only `transactionMetric` docs are being returned